src/test/resources/sng_book_various_content_rc/content/c01/01.usfm

```diff
@@ -0,0 +1,7 @@
+<para style="p">
```

Comment:
This looks like usx not usfm? where did you get it? Should usx be in a .usx file or are they allowed to share the usfm file extension?

Reply:
I copied it from the test resource
https://github.com/WycliffeAssociates/kotlin-resource-container/tree/dev/src/test/resources/valid_single_book_rc/content/01

Reply:
even worse; why did you need to duplicate a file you already have?

Reply:
I would say it is not very nice to have tests depend on a resource file within another test's sample file. I see that there are multiple rc which serving each test specifically